src/StatusQ/Controls/Validators/StatusAddressValidator.qml

```diff
@@ -0,0 +1,15 @@
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1
+
+StatusValidator {
+    name: "address"
+
+    error: qsTr("Please enter a valid address")
```

Comment:
So far we've avoided `qsTr` inside StatusQ because Status Desktop will replace `qsTr` with `qsTrId`.

We probably don't want Status Desktop to change StatusQ's source. So giving it plain text would be better here. Applications can then still i18n the message in whatever way they prefer (`qsTr` or other solutions)

Reply:
One more thought here:

How would you feel about changing this to `errorMessage` instead?

Reply:
Updated.